docs/source/Intro.rst

```diff
@@ -24,6 +24,13 @@ Challenges for accurate weather predictions still exist in numerical models. To
     - Larger TC False Alarm Rate in the western North Atlantic
     - Exacerbated low instability bias, which is largely related to dry soil moisture
     - Poor representation of radiation inversions
+  .. group-tab:: RRFS.v1alpha
+
+    - Better hurricane track forecasts for strong storms in the Atlantic basin
+    - Weak hurricane intensity forecasts
+    - Unstable signal over mountain area
+    - Slightly better cold air damming signal
+    - The heat wave signal is too strong
    
 This document provides case studies evaluation results for GFS using the `UFS Medium Range Weather (MRW) App <https://ufs-mrweather-app.readthedocs.io/en/latest/index.html>`_ with the target audience being a broad community. These cases are also run using the NOAA Environmental Model Center (EMC) `Global Workflow <https://github.com/NOAA-EMC/global-workflow/wiki>`_, which points to the most up-to-date GFS codes, and are expected to provide insights for the on-going model development. Brief descriptions for MRW App and Global Workflow are below. 
```

Comment:
I suggest deleting this paragraph. It is out of date, since SRW is not mentioned. Also, this information is in the introductory paragraph.

Reply:
It's deleted. Thanks!